Capm-domain3-agile-frameworks-guide.md

```
@@ -0,0 +1,497 @@
+CAPMでは「組織のプロセス資産(Organizational Process Assets, OPA)」と「組織体の環境要因(Enterprise Environmental Factors, EEF)」という用語が繰り返し登場します。ドメイン3では、これらのうちアダプティブアプローチの採用を後押しする要素を見分けられるかが問われます。
+
+| 区分 | 定義 | アダプティブ採用を促進する具体例 |
+|---|---|---|
+| OPA(組織のプロセス資産) | 組織が過去から蓄積してきた知識・手順・テンプレート・過去のプロジェクト記録など | アジャイルコーチングの社内ガイドライン、過去のスプリントレトロスペクティブの記録、既存のスクラムチームのベロシティデータ、社内のDefinition of Doneテンプレート |
+| EEF(組織体の環境要因) | 組織を取り巻く外部・内部の環境で、プロジェクトに影響を与えるがチームが直接コントロールできない要素 | 市場の変化スピードが速い業界文化、経営層のアジャイル推進方針、リモートワークを前提とした企業文化、変化に寛容な組織文化、アジャイルツール(Jira等)の全社導入状況 |
+
+> **ベストプラクティス**
+> - 予測型かアダプティブかを二択で捉えず、要求の不確実性・変更頻度・チームの専任度・組織文化という複数の軸で評価する。
+> - 組織構造がアダプティブに不向き(階層型・強いマトリックス)な場合でも、まずは小さなパイロットチームから始めて実績(OPA)を積み上げるアプローチが有効。
+> - EEFの評価では「変化を歓迎する文化があるか」「経営層のスポンサーシップがあるか」を必ず確認する。これが欠けているとアジャイル導入は形骸化しやすい。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)